check-coverage.sh

```diff
@@ -0,0 +1,10 @@
+LINES_COVERAGE_PERCENTAGE=`./node_modules/.bin/istanbul-combine -d coverage -p summary -r lcov -r html packages/frint*/coverage/coverage*.json | grep Lines | awk '{print $3}' | tr -d '[:space:]' | tr -d '%'`;
```

Comment:
could you kindly move this file to a generic directory, like `scripts`?

(based on how facebook/react has it in their repo for additional scripts)

Reply:
sure ... doing it now